cupy/creation/basic.py

```diff
@@ -55,6 +62,11 @@ def empty_like(a, dtype=None, order='K'):
             result. 'C' means C-order, 'F' means F-order, 'A' means 'F' if
             ``a`` is Fortran contiguous, 'C' otherwise. 'K' means match the
             layout of ``a`` as closely as possible.
+        subok: Not supported yet, must be None.
+        shape (tuple of ints): Overrides the shape of the result. If order='K'
```

Comment:
Should be `int or tuple of ints`.

Reply:
Sorry for the delay. You're right, I had that copied from the docs of `empty` (and similar) that don't include that information. I pushed a new commit to add that missing information to docs of all those functions now.